### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by kulnis182** [LAB02-Research/HASS.Agent#331]
CPU usage is completely off while reporting
For example. CPU usage is  12% or even higher but HASS ussually report as 1 or 2 %
Processor I7 10700K
Windows 11
![image](https://github.com/LAB02-Research/HASS.Agent/assets/77855355/f17483f8-e4d6-4da2-8027-9adb39b8efbd)

All other metric are correct

**Comment by amadeo-alex** [comment]
Hello, please use the bug report template and update the issue title to reflect "one sentence" issue description.
HASS.Agent uses performance counter for the CPU Load, for me it also gives "strange value" (value not matching what can be seen in the task manager.
This'll need more investigation/tests.

**Comment by amadeo-alex** [comment]
Yeah, looks like the PerformanceCounterSensor class is doing some rounding-shenanigans which cause the value to be off by a "bit":
![obraz](https://github.com/LAB02-Research/HASS.Agent/assets/68441479/4ec9682e-0c4c-438c-b09e-b92228868f2e)
![obraz](https://github.com/LAB02-Research/HASS.Agent/assets/68441479/691fd8fa-c6f9-4e8f-9ba0-67d9215dcfac)

Edit: scratch above, NextValue() is called two times right after another, will create PR to fix this in a sec.
